src/applications/caregivers/components/FormFields/FacilityList.jsx

```diff
@@ -48,13 +53,6 @@ const FacilityList = props => {
     />
   ));
 
-  useEffect(
```

Comment:
This was causing the page to jump to the top of the list when the `load more facilities` was loading more facilities. Now we let the parent component handle when to focus on the results.